tidb-cloud/recovery-group-get-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Get Started with Recovery Groups
+summary: Learn how to create a Recovery Group in TiDB Cloud and view its details
+---
+
+# Get Started with Recovery Groups
+
+This document describes how to create a Recovery Group to protect your databases running on TiDB Cloud Dedicated Clusters using the TiDB Cloud user interface. It also shows how to view details of the recovery group.
+
+## Prerequisites
+
+Recovery Groups replicate your databases to another cluster to protect your databases from a regional disaster. To create a recovery group you will need to have two TiDB Cloud Dedicated Clusters, one where the databases are primarily located, and a second cluster that will be used for the replicated copy of the databases. If you have not done so already, follow the steps in [Create a TiDB Dedicated Cluster](/tidb-cloud/create-tidb-cluster.md) to create the necessary clusters.
+
+> **Note**
+>
+> TiDB Cloud Recovery Groups are currently supported with TiDB Cloud Dedicated Cluster deployed on AWS
+
+## Create a new Recovery Group
+
+To create a Recovery Group, perform the following steps:
+
+1. In the TiDB Cloud console, click **Project Settings** in the left navigation pane.
+
+2. On the project settings navigation pane, click **Recovery Group**.
+
+3. On the recovery group page, click **Create Recovery Group**. A dialog is displayed.
+
+4. In the Create Recovery Group Dialog enter a name for the recovery group.
+
+    > **Note**
+    >
+    > Currently only one resiliency level is supported.
```

Comment:
Can you add explanation for what does resiliency level means for the users? What is the expected behavior for the only supported resiliency level?

Reply:
Add a new secton https://github.com/pingcap/docs/pull/17425/files#diff-ff23e50bb2b6fd16bac6cf66da63f4019cc00a42f986fba4ba08be01c0632725R74